core/client/hdfs/src/main/java/alluxio/hadoop/AlluxioHdfsFileSystem.java

```diff
@@ -119,7 +138,7 @@ public boolean exists(AlluxioURI alluxioURI, ExistsPOptions options) {
 
   @Override
   public AlluxioConfiguration getConf() {
-    return InstancedConfiguration.defaults();
+    return mAlluxioConf;
```

Comment:
we shall not return the default configuration but the real Alluxio config as `LocalCacheFileInStream` depends on this https://github.com/Alluxio/alluxio/blob/e488f4d48d6415e664203d5d3fc84e755ff328b8/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java#L102